Added client/server compiled models.

melezov opened the pull request.

melezov committed:
Added client transport model.
Added server persistence model.

dstimac commented on client/src/main/scala/com/mentatlabs/bepo/email/Attachment.scala:
```diff
@@ -0,0 +1,36 @@
+    Attachment(filename, MimeType.fromFileName(filename), content)
+}
+
+object MimeType extends App {
```
remove "extends App"

dstimac commented on client/src/main/scala/com/mentatlabs/bepo/email/Email.scala:
```diff
@@ -0,0 +1,47 @@
+  , htmlBody: Option[HtmlBody]
+  , attachments: Seq[Attachment]) {
+
+  def add(address: Address) = address match {
```
use-case add seq Address

dstimac commented on model/sql/db-update.bat:
```diff
@@ -0,0 +1,35 @@
+psql -Upostgres -q -c "DROP ROLE bepo;"
+
+echo Creating role ...
+psql -Upostgres -q -c "CREATE ROLE bepo WITH LOGIN SUPERUSER PASSWORD 'bepo';"
```
SUPERUSER -> NOSUPERUSER, pwwwease

melezov committed:
Client: Example event submission

dstimac commented:
+1

melezov committed:
Initial working version (client+server+mailer)
TODO: Google doesn't understand "related" nested under "alternative"
TODO: Dependancy on locally published revenj-servlet, needs a release
